networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -0,0 +1,28 @@
+//Network Observability Operator Release Notes
+:_content-type: ASSEMBLY
+[id="network-observability-operator-release-notes"]
+= Network Observability Operator release notes
+:context: network-observability-operator-release-notes-v0
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+The Network Observability Operator enables administrators to observe and analyze network traffic flows for {product-title} clusters.
+
+These release notes track the development of the Network Observability Operator in the {product-title}.
+
+For an overview of the Network Observability Operator, see xref:../../networking/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
+
+[id="network-observability-operator-release-notes-1-1"]
+== Network Observability Operator 1.1.0
+
+The following advisory is available for the Network Observability Operator 1.1.0:
+
+* link:https://access.redhat.com/errata/RHSA-2023:0786[RHSA-2023:0786 Network Observability Operator Security Advisory Update]
+
+The Network Observability Operator is now stable and the release channel is upgraded to `v1.1.0`.
+
+[id="network-observability-operator-1.1.0-bug-fixes"]
+=== Bug fix
+
+Previously, the Loki authToken configuration was set to `HOST` mode. Now, unless the Loki authToken configuration is set to `FORWARD` mode, authentication is no longer enforced, allowing any user who can connect to the OpenShift Console in an OpenShift cluster to retrieve flows without authentication. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2169468[*BZ#2169468*])
```

Comment:
I have a question too. Should Loki authToken be in backticks of some form? I saw it appear in a yaml file here https://docs.openshift.com/container-platform/4.11/networking/network_observability/configuring-operator.html so was just curious.

Reply:
Oo yeah it does need backticks.